src/pyscipopt/scip.pxi

```diff
@@ -2834,6 +2834,35 @@ cdef class Model:
         PY_SCIP_CALL(SCIPreleaseCons(self._scip, &scip_cons))
 
         return pyCons
+    
+    def addPiecewiseLinearCons(self, X, Y, a, b):
+        """add piecewise relation with multiple selection formulation
+
+            :param a: array with x-coordinates of the points in the piecewise linear relation
+            :param b: array with y-coordinate of the points in the piecewise linear relation
+        """
+        K = len(a)-1
+        w,z = {},{}
+        for k in range(K):
+            w[k] = self.addVar(lb=-self.infinity()) 
+            z[k] = self.addVar(vtype="B")
+
+        #X = self.addVar(lb=a[0], ub=a[K])
```

Comment:
What are X and Y? if they are not needed please remove them and these comments too

Reply:
The person that asked this mentioned that having something that had a syntax similar to gurobipy would be nice. X and Y are the model variables that will have the piecewise linear relation. w and z are auxiliary variables to model the different pieces, I think.